test/tests/serial/vmail-id.bats

```diff
@@ -41,35 +40,27 @@ function teardown_file() { _default_teardown ; }
 # This test case is shared with tests.bats, but provides context on errors + some minor edits
 # TODO: Could improve in future with keywords from https://github.com/docker-mailserver/docker-mailserver/pull/3550#issuecomment-1738509088
 # Potentially via a helper that allows an optional fixed number of errors to be present if they were intentional
-@test '/var/log/mail/mail.log is error free' {
+@test 'Mail log is error free' {
   # Postfix: https://serverfault.com/questions/934703/postfix-451-4-3-0-temporary-lookup-failure
-  _run_in_container grep 'non-null host address bits in' /var/log/mail/mail.log
-  assert_failure
+  _service_log_should_not_contain_string 'mail' 'non-null host address bits in'
 
   # Postfix delivery failure: https://github.com/docker-mailserver/docker-mailserver/issues/230
-  _run_in_container grep 'mail system configuration error' /var/log/mail/mail.log
-  assert_failure
+  _service_log_should_not_contain_string 'mail' 'mail system configuration error'
 
   # Unknown error source: https://github.com/docker-mailserver/docker-mailserver/pull/85
-  _run_in_container grep -i ': error:' /var/log/mail/mail.log
-  assert_failure
+  _service_log_should_not_contain_string 'mail' ': error:'
 
   # Unknown error source: https://github.com/docker-mailserver/docker-mailserver/pull/320
-  _run_in_container grep -i 'not writable' /var/log/mail/mail.log
-  assert_failure
-  _run_in_container grep -i 'permission denied' /var/log/mail/mail.log
-  assert_failure
+  _service_log_should_not_contain_string 'mail' 'not writable'
+  _service_log_should_not_contain_string 'mail' 'permission denied'
 
   # Amavis: https://forum.howtoforge.com/threads/postfix-smtp-error-caused-by-clamav-cant-connect-to-a-unix-socket-var-run-clamav-clamd-ctl.81002/
-  _run_in_container grep -i '(!)connect' /var/log/mail/mail.log
-  assert_failure
+  _service_log_should_not_contain_string 'mail' '\(!\)connect'
 
   # Postfix: https://github.com/docker-mailserver/docker-mailserver/pull/2597
-  _run_in_container grep -i 'using backwards-compatible default setting' /var/log/mail/mail.log
-  assert_failure
+  _service_log_should_not_contain_string 'mail' 'using backwards-compatible default setting'
```

Comment:
The referenced PR has similar looking log lines, but it seems that the check here was referring to [log lines shown in this comment](https://github.com/docker-mailserver/docker-mailserver/pull/2598#issuecomment-1141176633):

```suggestion
   # Log line match example: https://github.com/docker-mailserver/docker-mailserver/pull/2598#issuecomment-1141176633
  _service_log_should_not_contain_string 'mail' 'using backwards-compatible default setting'
```